draft-ietf-httpbis-priority.md

```diff
@@ -825,8 +826,8 @@ those coming from others.
 
 It is sometimes beneficial for the server running behind an intermediary to obey
 Priority header field values. As an example, a resource-constrained
-server might defer the transmission of software update files that would have the
-background urgency being associated. However, in the worst case, the asymmetry
+server might defer the transmission of software update files that have the
+background urgency. However, in the worst case, the asymmetry
```

Comment:
Not sure if this is the right fix, but something seemed off here and this is a proposal for it.

Reply:
Taking a closer look, this seems like text that survived from some early drafts and not something that got botched by any subsequent edits. @kaduk's suggestion here works for me.

Reply:
+1 to adopting the suggested change.